Purpose: Refer to /usr/share/common-licenses/GPL-2 and not the versionless
         symlink /usr/share/common-licenses/GPL.
Authour: Kel Modderman <kel@otaku42.de>
Fixes:   -
Status:  Debian specific

--- a/COPYRIGHT
+++ b/COPYRIGHT
@@ -15,5 +15,5 @@
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
 
 On Debian GNU/Linux systems, the complete text of the GNU General
-Public License can be found in `/usr/share/common-licenses/GPL'.
+Public License can be found in `/usr/share/common-licenses/GPL-2'.
 
